hoomd/md/integrate.py

```diff
@@ -102,8 +102,8 @@ class Integrator(_DynamicIntegrator):
     The following commands can be used to specify the integration methods used
     by integrate.mode_standard.
 
-    - :py:class:`hoomd.md.methods.brownian`
-    - :py:class:`hoomd.md.methods.Langevin`
+    - ``hoomd.md.methods.Brownian``
+    - ``hoomd.md.methods.Langevin``
```

Comment:
@syjlee Note that single vs double back ticks mean very different things in Sphinx/restructured text. Single backticks make a reference that links to the named object. Double backticks mark a section of verbatim text that should be formatted as code.